en/ability.json

```
@@ -193,83 +193,83 @@
   },
   "flameBody": {
     "name": "Flame Body",
-    "description": "Contact with the Pokémon may burn the attacker."
+    "description": "Contact with the Pokémon has a 30% chance to burn the attacker."
   },
   "runAway": {
     "name": "Run Away",
     "description": "Enables a sure getaway from wild Pokémon."
   },
   "keenEye": {
     "name": "Keen Eye",
-    "description": "Keen eyes prevent other Pokémon from lowering this Pokémon’s accuracy."
+    "description": "The Pokémon’s keen eyes prevent its accuracy from being lowered. It also ignores the target’s stages in evasion."
   },
   "hyperCutter": {
     "name": "Hyper Cutter",
-    "description": "The Pokémon’s proud of its powerful pincers. They prevent other Pokémon from lowering its Attack stat."
+    "description": "The Pokémon’s prized, mighty pincers prevent other Pokémon from lowering its Attack stat."
   },
   "pickup": {
     "name": "Pickup",
-    "description": "The Pokémon may pick up the item an opposing Pokémon held during a battle."
+    "description": "The Pokémon will attempt to steal an item that an opposing Pokémon held after a battle is won, even if it did not participate."
   },
   "truant": {
     "name": "Truant",
-    "description": "The Pokémon can’t use a move if it had used a move on the previous turn."
+    "description": "Each time the Pokémon uses a move, it spends the next turn loafing around."
   },
   "hustle": {
     "name": "Hustle",
-    "description": "Boosts the Attack stat, but lowers accuracy."
+    "description": "Boosts the Pokémon’s Attack stat by 50%, but lowers the accuracy of physical moves by 20%."
   },
   "cuteCharm": {
     "name": "Cute Charm",
-    "description": "Contact with the Pokémon may cause infatuation."
+    "description": "Contact with the Pokémon has a 30% chance to infatuate the attacker."
```

```suggestion
    "description": "The Pokémon has a 30% chance to infatuate attackers that make direct contact with it."
```
For some reason only this and Static had their descs updated in gen 9